src/Microsoft.Azure.WebJobs.Host/SharedListenerAttribute.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host
+{
+    /// <summary>
+    /// Attribute applied to a <see cref="ITriggerBinding"/> implementation when the binding uses
+    /// a shared listener for all functions.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class)]
+    public class SharedListenerAttribute : Attribute
```

Comment:
This attribute must be applied to the ITriggerBinding implementation of the extension using a shared listener.